src/__fixtures__/Events/37-entity-in-title.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "entity in title (#592)",
+    "html": "<title>the &quot;title&quot;",
+    "expected": [
+        {
+            "event": "opentagname",
+            "data": ["title"]
+        },
+        {
+            "event": "opentag",
+            "data": ["title", {}]
+        },
+        {
+            "event": "text",
+            "data": ["the \"title\""]
+        },
+        {
+            "event": "closetag",
```

Comment:
is this event actually fired? I don't see a `</title>` in the html string for this fixture

Reply:
The parser will close all currently open tags at the end of the input.